build.gradle

```diff
@@ -247,7 +250,7 @@ subprojects {
         if (project.hasProperty("shouldDeleteSchema") && Boolean.valueOf(project.shouldDeleteSchema.toString())) {
             task.dependsOn(deleteSchema)
         }
-        task.finalizedBy(scalaStyle)
+        //task.finalizedBy(scalastyleCheck)
```

Comment:
Unfortunately, the newer scala style plugin looks like it doesn't support specifying source configuration: https://github.com/alisiikh/gradle-scalastyle-plugin#configuration so running this would fail on our generated code. Maybe we can put in a feature request against the repo?

Reply:
hmm. are we able to keep using the old one since its only a buildscript dependency? maybe the old one is able to run against 2.13 source code

Reply:
i guess we could also move the generated code to a separate `generated` source set, it looks like the new plugin supports excluding a source set

Reply:
I tested the old one, it's able to run against 2.13 source code. We can add a note for that, or we can move to the separated source set. I'm open to either one

Reply:
hmm, lets keep using the old one for now since its able to run against 2.13 source code